style.css

```diff
@@ -49,10 +73,10 @@ body {
   background-color: #686868;
 }
 
-.number-card p {
-  padding-left: 60px;
-  max-width: fit-content;
+.number-card {
```

Comment:
you already have this selector on line 65. which is also missing a line break before it